rasgotransforms/rasgotransforms/transforms/line_chart/line_chart.yaml

```diff
@@ -8,6 +8,9 @@ arguments:
   axis:
     type: column
     description: axis column
+  axis_is_date:
```

Comment:
Is there a way to make this optional and default to FALSE?

Reply:
>I'm going to make this pencil disappear